src/View3d.ts

```diff
@@ -977,29 +978,54 @@ export class View3d {
    * be in the normalized coordinate space of the Volume, where the origin
    * (0,0,0) is at the center of the Volume and the extent is from -0.5 to 0.5
    * in each axis.
+   * @deprecated Will be removed in the next major release. Use `addDrawableObject` instead.
    */
   addLineObject(line: Line3d): void {
-    if (this.image) {
-      this.image.addLineObject(line);
-      this.redraw();
-    }
+    return this.addDrawableObject(line);
```

Comment:
I am wondering if making these more generic/polymorphic makes it harder to reorder them for specific rendering needs.  just a random thought.